civihr_default_theme/scss/base/_variables.scss

```diff
@@ -118,6 +118,7 @@ $nav-tabs-justified-active-link-border-color: $light !default;
 
 $screen-sm-max:                   767px !default;
 $crm-menu-responsive-breakpoint:  1250px;
+$crm-onboarding-wizard-menu-responsive-breakpoint:  1300px;
```

Comment:
if this variable is used only for the onboarding wizard, why define it in the base variables file?

Reply:
Here we have all the responsive variables defined, just like `$crm-menu-responsive-breakpoint`. So kept all at the same place.